backend/src/main/java/com/redhat/cloud/notifications/recipients/rbac/AuthRequestFilter.java

```diff
@@ -26,6 +27,7 @@
     // used by dev to by pass the service to service token: Uses user:password format
     static final String RBAC_SERVICE_TO_SERVICE_DEV_EXCEPTIONAL_AUTH_KEY = "rbac.service-to-service.exceptional.auth.info";
 
+    @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
wouldn't setting this on true lead to other issues when a field changes again? I mean, could it be that they add an important field and we don't get it's not in our class and we ignore unknown properties? Is this a case we should think about?

Reply:
We only use the `secret` field in the code. If this one disappears, then the app won't start correctly and we'll know immediately that something's wrong. I don't think ignoring new fields will cause any issue in the future.

Reply:
ok.